modules/@apostrophecms/doc/index.js

```diff
@@ -1752,14 +1751,10 @@ module.exports = {
               }
             } else if (field.type === 'object') {
               const value = doc[field.name];
-              if (value) {
-                handlers.object(field, value);
-                forSchema(field.schema, value);
-              }
+              handlers.object(field, value);
+              forSchema(field.schema, value);
```

Comment:
Same as below for consistency

Reply:
@ValJed should we do the same also for arrays above?

Reply:
well arrays are different

Reply:
I wanted yes but the problem is that for arrays we call `arrayItem` and not `array` handler. It is called on each array item and not on the array itself.
I could have change that maybe and passed directly the array?
But I wanted to bring the less amount of changes possible.